gix-blame/src/file/function.rs

```diff
@@ -666,6 +688,13 @@ fn collect_parents(
     Ok(parent_ids)
 }
 
+fn tree_id(commit: gix_traverse::commit::Either<'_, '_>) -> Result<ObjectId, Error> {
```

Comment:
I could imagine providing this function as method on `Either` one day, which by then could probably also be renamed in something more specific.